www-apps/davical/davical-1.1.7-r2.ebuild

```diff
@@ -1,27 +1,28 @@
-# Copyright 1999-2018 Gentoo Authors
+# Copyright 1999-2019 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
```

Comment:
And in these, don't use `git mv` but make a new copy for every new file. Ie, separate addition and deletion.

Reply:
you mean I should move the deletion into another commits, right?